unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/CollidersManager/CollidersManager.cs

```diff
@@ -168,5 +168,35 @@ public void ConfigureColliders(GameObject meshGameObject, bool hasCollision, boo
                 }
             }
         }
+        
+        public void CreateColliders(GameObject meshGameObject, MeshFilter[] meshFilters, bool hasCollision, bool isPointerBlocker, IDCLEntity entity, int layerMask)
+        {
+            if (meshGameObject == null || (!hasCollision && !isPointerBlocker))
+                return;
+            
+            entity.meshesInfo.colliders.Clear(); 
+            
+            // This is de default layer, however, take into account that OnPointerEvents can change this layer
+            int colliderLayer = layerMask;//LayerUtils.CalculateLayerMask(entity.entityId, hasCollision, isPointerBlocker);
```

Comment:
We should remove the `//LayerUtils.CalculateLayerMask(entity.entityId, hasCollision, isPointerBlocker);` part right?